lang/luasec/patches/100-fix-compilation.patch

```diff
@@ -1,7 +1,16 @@
-diff --git a/src/Makefile b/src/Makefile
-index 9be2f14..93d1dc4 100644
 --- a/src/Makefile
 +++ b/src/Makefile
+@@ -15,8 +15,8 @@ WARN=-Wall -pedantic
+ BSD_CFLAGS=-O2 -fPIC $(WARN) $(INCDIR) $(DEFS)
+ BSD_LDFLAGS=-O -fPIC -shared $(LIBDIR)
+ 
+-LNX_CFLAGS=-O2 -fPIC $(WARN) $(INCDIR) $(DEFS)
+-LNX_LDFLAGS=-O -fPIC -shared $(LIBDIR)
++LNX_CFLAGS=$(INCDIR) $(DEFS)
++LNX_LDFLAGS=-O -shared $(LIBDIR)
```

Comment:
```suggestion
+LNX_LDFLAGS=-O1 -shared $(LIBDIR)
```
The Makefile calls `ld` directly, where the `-O` option requires an argument.  Right now `-shared` is that argument (it used to be `-fPIC`), so it gets ignored, and build fails with 
```
mipsel-openwrt-linux-musl-ld -L/home/equeiroz/src/openwrt-asus/staging_dir/target-mipsel_74kc_musl/usr/lib -L/home/equeiroz/src/openwrt-asus/staging_dir/target-mipsel_74kc_musl/lib -L/home/equeiroz/src/openwrt-asus/staging_dir/toolchain-mipsel_74kc_gcc-7.4.0_musl/usr/lib -L/home/equeiroz/src/openwrt-asus/staging_dir/toolchain-mipsel_74kc_gcc-7.4.0_musl/lib -znow -zrelro -fpic   -O -shared -L/home/equeiroz/src/openwrt-asus/staging_dir/target-mipsel_74kc_musl/usr/lib -L/home/equeiroz/src/openwrt-asus/staging_dir/target-mipsel_74kc_musl/lib -L/home/equeiroz/src/openwrt-asus/staging_dir/toolchain-mipsel_74kc_gcc-7.4.0_musl/usr/lib -L/home/equeiroz/src/openwrt-asus/staging_dir/toolchain-mipsel_74kc_gcc-7.4.0_musl/lib -znow -zrelro -fpic -L./luasocket -o ssl.so x509.o context.o ssl.o config.o ec.o -lssl -lcrypto -lluasocket
mipsel-openwrt-linux-musl-ld: -f may not be used without -shared
```
I'm not even sure we should use optimization with ld here, as we're not using it in general.

Reply:
I don't think so either. Removed.